bilby/gw/detector/calibration.py

```diff
@@ -82,10 +110,33 @@ def write_calibration_file(filename, frequency_array, calibration_draws, calibra
         Shape is (number_of_response_curves x len(frequency_array))
     calibration_parameter_draws: data_frame
         Parameters used to generate the random draws of the calibration response curves
+    correction: str
+        How the correction is defined, either to the data (default) or
+        the template, see :func:`bilby.gw.prior.CalibrationPriorDict.from_envelope_file`.
+        If :code:`data`, the calibration draws will be inverted before being saved so that the format
+        matches files produced by detector calibration pipelines.
 
     """
     import tables
 
+    if correction is None:
```

Comment:
There is some code duplication here. These statements are used three times in total. Would it be possible to wrap this to a function like `_check_correction_type(correction)` or something similar?